File: Examples/Features/Workbook/SaveWorkbookToExcelFile.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Workbook
{
    public class SaveWorkbookToExcelFile : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            //change the path to real export path when save.
            workbook.Save(this.CurrentDirectory + "dest.xlsx", SaveFileFormat.Xlsx);

        }

        public override bool CanDownload
        {
            get
            {
                return false;
            }
        }

        public override bool ShowViewer
        {
            get
            {
                return false;
            }
        }

        public override bool IsUpdate
        {
            get
            {
                return true;
            }
        }
    }
}

File: Examples.VB/Features/Workbook/SaveWorkbookToExcelFile.vb
Namespace Features.Workbook
    Public Class SaveWorkbookToExcelFile
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            'change the path to real export path when save.
            workbook.Save(IO.Path.Combine(CurrentDirectory, "dest.xlsx"), SaveFileFormat.Xlsx)
        End Sub
        Public Overrides ReadOnly Property CanDownload As Boolean
            Get
                Return False
            End Get
        End Property
        Public Overrides ReadOnly Property ShowViewer As Boolean
            Get
                Return False
            End Get
        End Property
        Public Overrides ReadOnly Property IsUpdate As Boolean
            Get
                Return True
            End Get
        End Property
    End Class
End Namespace
